### DISCUSSION TITLE
synch farm and block farm to make 1 filter pass 1

### DISCUSSION THREAD

**Discussion by mkdev-code** [Chia-Network/chia-blockchain#5516] [Support]
synch farm and block farm to make / 1 filter pass 1 time unknown wining same

33/16/8 years for win 
8h for 1 filter pass ?  then 4h for 2 filters ... ? 
some win at 2 plots, some at 20 plots,  some with more wins more ? luck 0

using 6 plots k32
block farmed 0 so what time will be for  1 block to be farmed even with no win ? time

**Comment by mkdev-code** [comment]
faq
bit explains it 
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-plot-filter-and-why-didnt-my-plot-pass-it

https://imgur.com/T9fRTkp

What is the plot filter and why didn't my plot pass it?
Farmers compute a plot filter based on the signage point, their plot id, and the sub-slot challenge - which are hashed together to create the plot filter bits. If the plot filter bits start with 9 zeroes, that plot passes the filter for that signage point, and can proceed. This disqualifies around 511/512 of all proofs of space on the network, for each signage point. There are 4608 * 2 or 9216 signage points per day so the average plot should pass the filter 18 times per 24 hours on average. Once a plot passes the plot filter it then competes for the best proof of space with every other plot that also passed that plot filter for that signage point. For reasons that aren't super simple to intuit, the only thing each plot is competing on is to have the best proof of space and thus the chances of getting a reward depend on total size of plots on the farm - even with the plot filter in place.

As long as the plot passes the filter, and do not have any internal file errors, the plot will always be eligible to compete for the best proof of space. Moving the plot to another directory or server will not change its eligibility.

Please note that the speed of your lookups when passing the plot filter should be below 5 and preferably below 2 seconds. When you actually win a reward, your drive will have to do more lookups than these, so it's important that the the lookups are happening fast.




so chance is 2 plot to be not ok   , but 1 week still no block chance is low even not to win ?
As long as the plot passes the filter, and do not have any internal file errors, the plot will always be eligible to compete for the best proof of space. Moving the plot to another directory or server will not change its eligibility.

**Comment by BrandtH22** [comment]
@mkdev-code , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
